### ISSUE TITLE
Dynamic variables from PAW are not working in Runscope

### ISSUE THREAD

**Issue by tettomagic** [Runscope/paw-runscope#3]
When I create a dynamic variable in PAW, it's not getting exported to Runscope as responses variables. I have to redefine the variable again in Runscope and so on.

**Comment by mittsh** [comment]
@johnsheehan Is it something that makes sense for Runscope variables? If so that should be an easy fix.

**Comment by johnsheehan** [comment]
Our import doesn't support environment variables. We could make the steps include the `{{}}` placeholders, but the actual values would need to be manually entered in the environment after importing the steps.

**Comment by TrentBrown** [comment]
I sure would like to be able to do this, too. The fact that Paw runs only on the Mac means that we have to provide another way for customers to play with running workflows against our API.

**Comment by mittsh** [comment]
@TrentBrown We have a great project coming to let users integrate Paw files with other platforms easily… If you're looking for a way to let your API users play with your API in any platform or client, that's exactly what we're working on. Ping me at micha AT paw [DOT] cloud I'll send you some steps.

**Comment by TrentBrown** [comment]
Hi Micha,

Interesting. That sounds like it might be useful for our purposes.

One of the things that I particularly like about Paw is that I can create a document that refers to secrets (eg. API key and secret) by name, outside of the document, like the Apple Keychain. That prevents my accidentally leaving my own keys in the document when I send a copy of it to our customers. Will your web-based app have a way to do something similar? Regardless of how carefully you might encrypt my keys, I would not want to store them on your system.


Thanks,

       —Trent

**Comment by mittsh** [comment]
Absolutely, whatever we do we will continue to support the ability to keep user credentials local (and encryption of stored credentials) @TrentBrown
